client/search-ui/src/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>SearchUi</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+</head>
+<body>
+  <app-root></app-root>
```

Comment:
keep the appName as `google-search-app`

Reply:
I think, it should not be named as google-search-app as this project doesn't represent of google. It would be good to keep a generic name, that is search-app.

Thanks for your feedback in advance.

Reply:
never mind. but this app is google clone app